cmd/clusterctl/client/cluster/template.go

```diff
@@ -210,6 +215,21 @@ func (t *templateClient) getGitHubFileContent(rURL *url.URL) ([]byte, error) {
 	return content, nil
 }
 
+func (t *templateClient) getRawURLFileContent(rURL string) ([]byte, error) {
+	res, err := t.httpClient.Get(rURL)
```

Comment:
Why the lint showing `Get must not be called`? Without Get we can't fetch the response right? Is there any workaound we can do?

Reply:
Should be done ~ like this:

```go
    // Change NewRequest to NewRequestWithContext and pass context it
    req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://example.com", http.NoBody)
    ...

    _, err = http.DefaultClient.Do(req)
    ...
```

P.S. You can usually google those linter findings => https://github.com/sonatard/noctx

Reply:
Thanks!